src/view/handlebars.rs

```diff
@@ -0,0 +1,99 @@
+use response::{Serializer, SerializerContext, ContentType};
+
+use bytes::Bytes;
+use handlebars::Handlebars as Registery;
+use http::header::HeaderValue;
+use serde::Serialize;
+
+use std::env;
+use std::path::Path;
+use std::sync::Arc;
+
+/// Serialize response values using Handlebars templates
+///
+/// This serializer is able to render handlebar templates using structs with
+/// `#[derive(Response)]` and a template name, set with the `#[web(template =
+/// "<template name>")]` annotation.
+#[derive(Clone, Debug)]
+pub struct Handlebars {
+    registery: Arc<Registery>,
+    html: HeaderValue,
+}
+
+const TEXT_HTML: &str = "text/html";
+
+impl Handlebars {
+    /// Create a new handlebars serializer.
+    ///
+    /// The serializer renders handlebar templates using the response value to
+    /// populate template variables. The response value must have
+    /// `#[derive(Response)]` and a template name specified using the
+    /// `#[web(template = "<template name>")]`.
+    ///
+    /// Templates are loaded from the `templates` directory in the crate root
+    /// and have the `.hbs` file extension.
+    pub fn new() -> Handlebars {
+        let mut registery = Registery::new();
```

Comment:
Ugh, you have got to be kidding me. The typo is in the library?

Reply:
Yeah no... all me. My keyboard loved to add `e`s everywhere and I use vim's autocomplete a lot... so, pretty much fail. (Also, I shipped off my laptop to have the keyboard repaired finally).